### ISSUE TITLE
Name is misleading / At least add screenshots/information of it working in previous Windows versions

### ISSUE THREAD

**Issue by spookyahell** [jithurjacob/Windows-10-Toast-Notifications#48]
I'm sort of missing the fact that this even works on Windows 7 in the README....
that's not a total obvious fact...

**Comment by na-stewart** [comment]
Python library to display Windows 10 Toast Notifications

**Comment by l-the-h** [comment]
> Python library to display Windows 10 Toast Notifications

LOL - ?? Why are you posting the description of the library essentially two years after I posted the issue, how is this helpful?

**Comment by frak0d** [comment]
yeah this is not even a toast library, its a baloon tip library which in windows 10 looks like toast notifications
